util/topgen/c_test.py

```diff
@@ -126,7 +126,7 @@ def _get_alert_peripherals(self):
                 continue
 
             # RV_DM module does not have DIF.
-            if inst_name == "rv_dm":
+            if entry['type'] == "rv_dm" or entry['type'] == "rv_dm_v1":
```

Comment:
Hm, is there a cleaner solution than to hard-code the version suffix here? Maybe
```suggestion
            if entry['type'].startswith("rv_dm")
```
?

Reply:
A "cleaner solution" would probably add a property to the hjson files and have that control what topgen generates, instead of hard-coding exceptions with specific named IP types :wink:

Reply:
Hehe, agreed. @Razer6 would you have time to look into this?

Reply:
I thought about the .startswith() but decided against it to be explicit.

What about adding a new property .generate_dif, set that to true of not specified, and set that to false for the RV_DM instances?

Reply:
SGTM!

Reply:
So now it'd be this, right?

```suggestion
            if entry['generate_dif']:
```

Reply:
Oh no. I missed saving the file. I think it is:

```python
if not entry['generate_dif']:
    continue
```

Reply:
There is still some diff generated, which I need to investigate. Will be a bit async since I'm on vacation this week.

Reply:
Seems the valie from the hjson is not picked as a bool but as a string.